common/src/main/java/com/microsoft/identity/common/internal/eststelemetry/PublicApiId.java

```diff
@@ -105,4 +105,12 @@
     public static final String MULTIPLE_ACCOUNT_PCA_REMOVE_ACCOUNT_WITH_ACCOUNT_CALLBACK = "930";
 
     //endregion
+
+    // region Device Code Flow
+
+    // PublicClientApplication
+    //==============================================================================================
+    public static final String DEVICE_CODE_FLOW_CALLBACK = "650";
```

Comment:
nit: rename to `DEVICE_CODE_FLOW_WITH_CALLBACK` to follow the same pattern as other consants